trie/iterator.go

```diff
@@ -23,18 +23,18 @@ import (
 // Print prints the trie through pre-order traversal
 func (t *Trie) Print() {
```

Comment:
Can we refactor the print statements to utilize log15

Reply:
ok what package is that?

Reply:
https://github.com/inconshreveable/log15

Reply:
are we using it anywhere already?

Reply:
on my branch `dustin-test/entrypoint`, its also used throughout go-ethereum if you're looking for examples.

Reply:
done